runtime/development/src/xcm.rs

```diff
@@ -319,9 +320,36 @@ pub type LocationToAccountId = (
 	// Sibling parachain origins convert to AccountId via the `ParaId::into`.
 	SiblingParachainConvertsVia<Sibling, AccountId>,
 	// Straight up local `AccountId32` origins just alias directly to `AccountId`.
+	// TODO(nuno): We probably should drop this because this allows for the relay-chain
```

Comment:
Will follow up on a separate issue

Reply:
@mustermeiszer I looked into this and it's not what we thought. We were afraid this would allow any XCM signed through the Relaychain to become a native account but that's not what's happenig. What this does is that if the origin of the XCM is a local account (i.e, within `Parachain(Centrifuge)`) then it aliases it to a native account directly, which makes sense. Just double checked and all the other parachains remain on using it as well